boards/nxp/mimxrt1180_evk/mimxrt1180_evk.dtsi

```diff
@@ -55,20 +55,51 @@
 	status = "okay";
 };
 
-&flexspi1 {
+&flexspi {
+	pinctrl-0 = <&pinmux_flexspi1>;
+	pinctrl-names = "default";
+};
+
+&flexspi {
 	status = "okay";
 	ahb-prefetch;
 	ahb-read-addr-opt;
 	rx-clock-source = <1>;
 	w25q128jw: w25q128jw@0 {
```

Comment:
```suggestion
	w25q128jw: w25q128jw@2 {
```
Reg address should match here, otherwise we'll get a DTC warning

Reply:
Updated. May I ask why reg address is 2?

Reply:
Reg address follows the port the flash device is on. Address 2 indicates FLEXSPI port B1. Looking at the RT1180 schematic, this should actually be on port A1, right? So the reg address should be 0

Reply:
then these are defined wrong in the DT binding, each bus controller should be a node, port A should be a spi bus and port B should be a spi bus, each with their own node, not a problem for this PR though

Reply:
FLEXSPI has 4 ports A1,A2,B1,and B2 which can all operate independently. A1/B1 and A2/B2 can also be grouped to operate in parallel mode, so that a read/write to flash A1 will be split/merged across flash A1 and B1. Given this I'm not sure why we would group the ports as A1/A2 and B1/B2- is there something I'm missing?

Reply:
Any update about flash device address implication under flexspi module?